.github/workflows/windows-package.yml

```diff
@@ -0,0 +1,36 @@
+# This workflow will install compilers and packges on windows platform
+
+name: windows package
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master, develop ]
+
+name: windows
+jobs:
+  windows:
+    runs-on: windows-2019
+    strategy:
+      matrix:
+        compiler: [gcc, gcc-fortran]
+    defaults:
+      run:
+        shell: msys2 {0}
+    steps:
+    - uses: actions/checkout@v2
+    - uses: msys2/setup-msys2@v2
+      with:
+        install: >-
+          gcc make
+          gcc-fortran make
+          make mingw-w64-i686-winpthreads-git
+          make mingw-w64-i686-openblas
+    - run: pacman --noconfirm -S gcc
```

Comment:
I think the previous command will install gcc etc., automatically, so you don't need to run pacman separately. You can delete all `pacman` commands